src/spatial_ref/srs.rs

```diff
@@ -580,12 +567,8 @@ impl SpatialRef {
         let child = child.try_into().expect("`child` must fit in `c_int`");
 
         let c_node_path = CString::new(node_path)?;
-        let rv = unsafe { gdal_sys::OSRGetAttrValue(self.0, c_node_path.as_ptr(), child) };
-        if rv.is_null() {
-            Ok(None)
-        } else {
-            Ok(Some(_string(rv)))
-        }
+        let c_ptr = unsafe { gdal_sys::OSRGetAttrValue(self.0, c_node_path.as_ptr(), child) };
```

Comment:
`c_ptr` is owned by `c_node_path` here, so we don't need to free it?

Reply:
Yes, owned by the `OGR_SRSNode`.